FP_Factions/gordon_reskins/concealedvests/config.cpp

```diff
@@ -0,0 +1,95 @@
+class CfgPatches
+{
+	class gordon_reskins_modvest
+	{
+		// List of units defined in this "PBO"
+		units[]={};
+		weapons[]=
+		{
+			"FP_GOR_V_TacVest_inv",
+			"FP_GOR_V_TacVest_inv_space",
+			"FP_GOR_V_PlateCarrier2_inv", 
+			"FP_GOR_V_PlateCarrier2_inv_space"
+		};
+		requiredVersion=0.1;
+		requiredAddons[]=
+		{
+			"A3_Data_F_AOW_Loadorder"
+		};
+	};
+};
+class CfgWeapons
+{
+	class V_TacVest_oli;
+	class FP_GOR_V_TacVest_base: V_TacVest_oli
+	{
+		scope = 0;
+		class ItemInfo;
+	};
+	class FP_GOR_V_TacVest_inv_space: FP_GOR_V_TacVest_base
+	{
+		author = "Gordon Weedman";
+		_generalMacro = "V_TacVest_oli";
+		scope = 2;
+		displayName = "Concealed kevlar vest (Carry space)";
+		hiddenSelectionsTextures[]={""};

```

Comment:
would rather like to see a fully transparent 4x4 paa instead of an empty string. An empty string is actually interpreted as a faulty config/texture and thus it is not rendered.

Reply:
arma has one, that is 32x32 `a3\data_f\clear_empty.paa`
```suggestion
		hiddenSelectionsTextures[]={"a3\data_f\clear_empty.paa"};

```

Reply:
Probably a better solution is to use `\A3\Weapons_F\empty.p3d` as model and set the hiddenSelectionsTextures array to an empty array